.env.example

```diff
@@ -50,6 +50,10 @@ NMDC_GCS_USE_FAKE=True
 NMDC_GCP_PRIMARY_POSTGRES_URI_SECRET_ID=changeme
 NMDC_GCP_SECONDARY_POSTGRES_URI_SECRET_ID=changeme
 
+# == Google Map API Key ==
+# the google map api key is currently used to pass to geoloc tool to get elevation data. This key must have some value in it to run tests
+GOOGLE_MAP_API_KEY=thiscanbeanythingfortesting
```

Comment:
Can you add the `NMDC_` prefix to this variable name? It works as-is because we have `extra="allow"` in the settings config: https://github.com/microbiomedata/nmdc-server/blob/e1a29e1dbbfb16d75c856151b115b85690fd0704/nmdc_server/config.py#L201-L205

But having the prefix is a good indicator that it's a variable used in the `nmdc-server` codebase.